docs/howto/kubernetes/gardener/create-shoot-cluster.md

```diff
@@ -0,0 +1,111 @@
+→ [Managed Kubernetes](https://{{config.extra.gui_domain}}/containers/gardener).
+You will see a Gardener page, in which you can create and manage your clusters. Click `Create Kubernetes cluster`.
+
+![Gardener page in {{config.extra.gui}}](assets/gardener_page.png)
+
+> In Gardener's terminology, a Kubernetes cluster is referred as a **Shoot cluster**. You can see the name "Shoot" in various places throughout
+> the panel's UI, so it is good to know what it means. To learn more, check
+> out [Gardener architecture](https://gardener.cloud/docs/gardener/concepts/architecture/).
+
+In the opened form, fill in the name of the cluster and a region to see the rest of the options. Hover over question mark icons at each form
+field to learn more about it. For the purposes of this how-to, you can leave everything at default values and click `Create` at the bottom.
+
+> In the form you can see a section about worker groups. This name refers to Kubernetes worker nodes.
+
+In the list of clusters, you will see your new Gardener shoot bootstrapping. The icon on the left marks the progress. Creating the cluster
+can take up to several minutes.
+
+![Shoot cluster bootstrapping](assets/shoot_bootstrapping.png)
+
+## Extract kubeconfig file from Shoot cluster
+
+When the Shoot cluster is up and running, you need to get a kubeconfig file to be able to access it. To do that, click on the cluster to
+expand its properties, and open `Kubeconfig`.
+
+![Kubeconfig in Gardener Shoot view](assets/shoot_kubeconfig.png)
+
+You should now see contents of the file. You have the option to `Copy Config` or `Rotate Kubeconfig` if your credentials got compromised.
+Copy the content of the kubeconfig and insert it into `~/.kube/config`. Create the directory and the file if needed.
+
+> By default, Kubectl searches for config in `~/.kube/config`, but you can modify this behaviour if needed. More
+> info [here](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/).
+
+Check if your kubectl uses the proper configuration by running:
+
+```shell
+kubectl config view
+```
+
+You should see something like this:
+
+```yaml
+apiVersion: v1
+clusters:
+  - cluster:
+      certificate-authority-data: DATA+OMITTED
+      server: https://api.test-cluster.p40698.staging-k8s.{{config.extra.gui_domain}}
+    name: shoot--p40698--test-cluster
+contexts:
+  - context:
+      cluster: shoot--p40698--test-cluster
+      user: shoot--p40698--test-cluster-token
+    name: shoot--p40698--test-cluster
+current-context: shoot--p40698--test-cluster
+kind: Config
+preferences: { }
+users:
+  - name: shoot--p40698--test-cluster-token
+    user:
+      token: REDACTED
+```
+
+## Access your cluster with kubectl and deploy an application
+
+Check your available nodes by running:
+
+```shell
+kubectl get nodes
+```
+
+You should see Gardener's worker node that is available:
+
+```shell
```

Comment:
The [Pygments `shell` lexer](https://pygments.org/docs/lexers/#pygments.lexers.shell.BashLexer) is for shell *input;* the right lexer for shell *output* is [`console`](https://pygments.org/docs/lexers/#pygments.lexers.shell.BashSessionLexer).